src/gtimelog/main.py

```diff
@@ -1,1950 +1,49 @@
 """An application for keeping track of your time."""
-import time
-import sys
-
-
-DEBUG = '--debug' in sys.argv
-
-
-if DEBUG:
-    def mark_time(what=None, _prev=[0, 0]):
-        t = time.time()
-        if what:
-            print("{:.3f} ({:+.3f}) {}".format(t - _prev[1], t - _prev[0], what))
-        else:
-            print()
-            _prev[1] = t
-        _prev[0] = t
-else:
-    def mark_time(what=None):
-        pass
-
-
-mark_time()
-mark_time("in script")
-
-import collections
-import datetime
-import email
-import email.header
-import email.mime.text
-import functools
 import gettext
-import io
 import locale
-import logging
 import os
-import re
 import signal
-import smtplib
-from contextlib import closing
-from email.utils import parseaddr, formataddr
+import sys
 from gettext import gettext as _
-from io import StringIO
-
-mark_time("Python imports done")
 
+from gtimelog import DEBUG, root_logger
+from gtimelog.core.utils import mark_time
+from gtimelog.paths import LOCALE_DIR
 
-if DEBUG:
-    os.environ['G_ENABLE_DIAGNOSTIC'] = '1'
 
+def require_version(namespace, version):
```

Comment:
Wasn't this moved to another module in master very recently?

Reply:
My bad, bad rebase of my own. Returned to gtimelog/utils.py